enumero/core/src/test/scala/CaseEnumMacroSpec.scala

```diff
@@ -14,43 +12,39 @@ class CaseEnumMacroSpec extends WordSpec with Matchers {
     object Ale
   }
 
-  "@enum annotation" should {
-    "produce a valid CaseEnum-style ADT" in {
-      Planet.Earth shouldBe a[Product]
-      Planet.Earth shouldBe a[Serializable]
-      Planet.Earth shouldBe a[Planet]
-      Planet.Mercury should not be a[Planet.Earth.type]
-      Planet.Earth shouldBe Planet.Earth
-    }
+  test("@enum annotation should produce a valid CaseEnum-style ADT") {
+    assert(Planet.Earth.isInstanceOf[Product])
+    assert(Planet.Earth.isInstanceOf[Serializable])
+    assert(Planet.Earth.isInstanceOf[Planet])
+    assertEquals(Planet.Earth, Planet.Earth)
+  }
 
-    "produce a valid CaseEnum-style ADT (alternative syntax)" in {
-      Beer.Lager shouldBe a[Product]
-      Beer.Lager shouldBe a[Serializable]
-      Beer.Lager shouldBe a[Beer]
-      Beer.Ale should not be a[Beer.Lager.type]
-      Beer.Lager shouldBe Beer.Lager
-    }
+  test("@enum annotation should produce a valid CaseEnum-style ADT (alternative syntax)") {
+    assert(Beer.Lager.isInstanceOf[Product])
+    assert(Beer.Lager.isInstanceOf[Serializable])
+    assert(Beer.Lager.isInstanceOf[Beer])
+    assertEquals(Beer.Lager, Beer.Lager)
```

Comment:
we lost an assertion here

Reply:
it was intentional, when I tried to rewrite the assertion it wouldn't even compile ("fruitless type test") and it's such an obvious test that I preferred to drop it.